scss/modules/label.scss

```diff
@@ -1,35 +1,42 @@
 .husky-label-error,
 .husky-label-warning,
 .husky-label-success {
-    $height: 70px;
-
     width: 100%;
-    height: $height;
     position: relative;
     top: 0;
     left: 0;
     color: $white;
-    padding-left: 55px;
     font-size: $smallFontSize;
     z-index: 200;
 
+    &.big {
+        height: 70px;
```

Comment:
Thought about adding this one as default (meaning declaring it a level higher). Didn't really like it, I thought the label must always have a size class. Feels more consistent.